src/main/java/core/basesyntax/Car.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
-public class Car {
+public class Car implements Cloneable {
+    private Engine engine;
+    private String model;
+    private int mph;
+    private int capacity;
+    private int range;
 
+    public Car(Engine engine, String model, int mph, int capacity, int range) {
+        this.engine = engine;
+        this.model = model;
+        this.mph = mph;
+        this.capacity = capacity;
+        this.range = range;
+    }
+
+    @Override
+    public Car clone() throws CloneNotSupportedException {
```

Comment:
I guess  that "throws CloneNotSupportedException" has a grey color in your code. Cause actually this method doesn't throw that exception never. (Probably it has been left after you have tried to use super.clone() here which may throw that exception)